### ISSUE TITLE
qBittorrent integration does not work for qBittorrent 4.2

### ISSUE THREAD

**Issue by netvl** [idlesign/torrt#53]
Apparently qBittorrent changed their API in 4.1+, and older 4.0 and earlier methods, like querying `/login`, do not work any longer:
```
> torrt configure_rpc qbittorrent host=172.30.0.10 port=9099 user=<redacted> password=<redacted>
INFO: Configuring `qbittorrent` rpc ...
ERROR: Failed to query RPC `http://172.30.0.10:9099/login`:
ERROR: Failed to query RPC `login`:
ERROR: Failed to login using `http://172.30.0.10:9099/` RPC:
Traceback (most recent call last):
  File "/nix/store/i95ihm2jp5q0ngwjflw9hcihb5vgddk7-python3.7-torrt-0.14.0-dev/lib/python3.7/site-packages/torrt/rpc/qbittorrent.py", line 128, in query
    raise QBittorrentRPCException(response.text.strip())
torrt.rpc.qbittorrent.QBittorrentRPCException
```
New API is defined [here](https://github.com/qbittorrent/qBittorrent/wiki/Web-API-Documentation).

Note that qBittorrent 4.1.0, the one where they had changed the API, is more than two years old (released on May 4th, 2018), so maybe it would be better to upgrade, or at least add a compatibility note somewhere so people would know what to expect.

**Comment by idlesign** [comment]
Thank you for the report.
Will bear it in mind and provide a solution eventually.

**Comment by idlesign** [comment]
Switched to new API. Currently support is available in master. 
Compatibility note is added to README and docs.
Will be eventually released in torrt 0.15.0

Feel free to reopen if required.
